arelle/plugin/validate/ESEF/__init__.py

```diff
@@ -463,8 +463,8 @@ def checkFootnote(elt, text):
                         #    if scheme(href) in ("http", "https", "ftp"):
                         #        modelXbrl.error("ESEF.4.1.6.xHTMLDocumentContainsExternalReferences" if val.unconsolidated
                         #                        else "ESEF.3.5.1.inlineXbrlDocumentContainsExternalReferences",
-                        #            _("Inline XBRL instance documents MUST NOT contain any reference pointing to resources outside the reporting package: %(element)s"),
-                        #            modelObject=elt, element=eltTag,
+                        #            _("Inline XBRL instance documents MUST NOT contain any reference pointing to resources outside the reporting package: <%(element)s href=\"%(href)s\">"),
+                        #            modelObject=elt, element=eltTag, href=href,
```

Comment:
Above changes look fine, wondering why we are editing/keeping a commented out code block though